lib/octokit/client/apps.rb

```diff
@@ -15,30 +15,32 @@ def app(options = {})
         get 'app', options
       end
 
-      # Find all installations that belong to an App
+      # List all installations that belong to an App
       #
       # @param options [Hash] A customizable set of options
       #
       # @see https://docs.github.com/en/rest/apps/apps#list-installations-for-the-authenticated-app
       #
       # @return [Array<Sawyer::Resource>] the total_count and an array of installations
-      def find_app_installations(options = {})
+      def list_app_installations(options = {})
         paginate 'app/installations', options
       end
-      alias find_installations find_app_installations
+      alias find_installations list_app_installations
```

Comment:
Instead of aliasing, the other option is to create methods that warn of deprecation.